README.md

```diff
@@ -224,6 +224,25 @@ You can debug the main process by following one of these approaches:
 
 The inspector runs on port 9229
 
+### Linking with UI Libraries (e.g. React Polymorph)
+
+You can link libraries with Daedalus using one of the following steps:
+
+#### 1) Using `yalc`
+
+1) Install `yalc` globally using `yarn global add yalc`.
+2) Run `yalc publish` from the library's root directory that you want to link with Daedalus.
```

Comment:
I'd suggest to use a `yalc push` as otherwise `publish` requires a `--push` flag in order to have updates in Daedalus.